custom/icds_reports/static/js/icds_dashboard_utils.js

```diff
@@ -0,0 +1,33 @@
+/* global angular hqImport */
+
+function populateDashboardConstants() {
```

Comment:
I'm pretty sure this should be using hqDefine, though I was largely following the pattern used by the rest of the dashboard...